Bump docs-theme-extras to v0.1.21, add pipefail to framework-tests, added descriptions (#36)
agentregistry-dev/website: content/docs/agents/skills.md

---
title: Add skills

> description: "Give an agent access to skills published in agentregistry, loaded at runtime under the /skills folder."

Reviewer: ```suggestion
description: "Give an agent access to skills that are published in the registry catalog."
```
